web/ucp.php

```diff
@@ -43,6 +48,8 @@
         'langsArr' => Tools::getLangsArr(),
         'teamGroupsArr' => $teamGroupsArr,
         'templatesArr' => $templatesArr,
+        'visibilityArr' => $visibilityArr, // Added Visibility
```

Comment:
the comment should be removed, it sounds like a commit message ;)

Reply:
Should be solved in the latest changes